base_user_locale/models/res_company.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
```

Comment:
order  import, should be `fields, models`.

Reply:
done